### PR TITLE
Add status test

### PR DESCRIPTION
**Pull request by jmancewicz**

### CODE REVIEW

File: digits/test_status.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2014-2015, NVIDIA CORPORATION.  All rights reserved.
```
Comment (lukeyeager) [inline digits/test_status.py]: Should be `2015`

**Comment by lukeyeager** [comment]
This is excellent, thanks!

File: digits/test_status.py (hunk 1 above)
Comment (jmancewicz) [inline digits/test_status.py]: Changed. Thanks.

**Comment by lukeyeager** [comment]
It looks like `utils/time_filters.py` is causing the same kind of test coverage oscillation:

https://coveralls.io/builds/3463835

**Comment by jmancewicz** [comment]
I'll have a look.

**Pull request merged**